GRPCServiceA/pkg/entities/responses.go

```diff
@@ -0,0 +1,23 @@
+package entities
+
+type GetUserResp struct {
```

Comment:
same in previous try to place the files into the request and response files and packages. entities are used just  in domain business logic, into entities shouldn't be req o resp structs